src/main/java/com/employeesService/EmployeesService/repository/JpaEmployeeRepository.java

```diff
@@ -17,22 +19,24 @@
     private final EntityManager entityManager;
 
     @Autowired
-    public JpaEmployeeRepository(EntityManager entityManager){
-        this.entityManager=entityManager;
+    public JpaEmployeeRepository(EntityManager entityManager) {
+        this.entityManager = entityManager;
     }
 
     @Override
     @Transactional
     public Employee save(Employee employee) {
+        Salary salary=new Salary();
+        if (!employee.getSalaries().isEmpty()){
+            employee.getSalaries().add(salary);
+        }
```

Comment:
If it is not empty add it, otherwise? What if employ does not have any salary, you don't want to allow to add one?